--- node_modules/emojione/lib/js/emojione.js	2017-03-07 10:36:43.366301447 +0100
+++ node_modules/emojione/lib/js/emojione.js	2017-03-07 10:22:17.000000000 +0100
@@ -317,7 +317,7 @@
 
     ns.unicodeToImage = function(str) {
 
-        var replaceWith,unicode,short,fname,alt,title;
+        var replaceWith,unicode,shortName,fname,alt,title;
         var mappedUnicode = ns.mapUnicodeToShort();
         str = str.replace(ns.regUnicode, function(unicodeChar) {
             if( (typeof unicodeChar === 'undefined') || (unicodeChar === '') || (!(unicodeChar in ns.jsEscapeMap)) ) {
@@ -329,12 +329,12 @@
                 unicode = ns.jsEscapeMap[unicodeChar];
 
                 //then map to shortname and locate the filename
-                short = mappedUnicode[unicode];
-                fname = ns.emojioneList[short].fname;
+                shortName = mappedUnicode[unicode];
+                fname = ns.emojioneList[shortName].fname;
 
                 // depending on the settings, we'll either add the native unicode as the alt tag, otherwise the shortname
-                alt = (ns.unicodeAlt) ? ns.convert(unicode.toUpperCase()) : short;
-                title = ns.imageTitleTag ? 'title="'+short+'"' : '';
+                alt = (ns.unicodeAlt) ? ns.convert(unicode.toUpperCase()) : shortName;
+                title = ns.imageTitleTag ? 'title="'+shortName+'"' : '';
 
                 if(ns.imageType === 'png') {
                     if(ns.sprites) {
